### DISCUSSION TITLE
关于 UI 界面的思考🤔

### DISCUSSION THREAD

**Discussion by SONGjiemo** [xiaohucode/yidaRule#168] [Ideas]
1、视频显示排版
![image](https://github.com/xiaohucode/yidaRule/assets/46627639/7702090c-6edf-4e92-b761-6d9e876f868b)
应该默认是 网格三列。这样复合 16:9的手机图片缩略图。

2、跟多功能，应该在我的里面。因为这个地方是全局设置。
![image](https://github.com/xiaohucode/yidaRule/assets/46627639/6e4843de-ac26-4db5-8b5a-379d98923afd)

3、发现页面最好就是宫格显示，数据多了后就上下滑动不方便了。
![image](https://github.com/xiaohucode/yidaRule/assets/46627639/53dc41b0-dbba-4042-8003-727993fb45b3)

4、播放页的信息显示，最好就是  直接 3S 倒计时。我们普通用户都看不懂也不看这。
![image](https://github.com/xiaohucode/yidaRule/assets/46627639/da6df467-ba6e-4009-a3f5-452809fdceea)

5、搜索页面，勾选一个搜索类别。如果不勾选类别会导致搜索一个资源很久。
![image](https://github.com/xiaohucode/yidaRule/assets/46627639/32519ced-2140-42fc-960b-bfa519089cdf)

6、自动暂时关闭站点，检测到报错。或者添加一个一件检测全部站点功能。把失效的坚持后默认都关闭。
![image](https://github.com/xiaohucode/yidaRule/assets/46627639/ad49ce58-57c6-4318-a6f5-b15ee2cf217a)

**Comment by ljun223** [comment]
老哥你是干产品经理的吗？😂

**Reply by SONGjiemo** [reply]
被你猜到了😂😂😂

**Comment by xiaohucode** [comment]
> 1、视频显示排版 

ok

>  2、跟多功能，应该在我的里面。因为这个地方是全局设置。 

ok

>  3、发现页面最好就是宫格显示，数据多了后就上下滑动不方便了。

是这样么？

<img src="https://github.com/xiaohucode/yidaRule/assets/24434389/8f0a4601-59ac-420b-b919-c55417f10738">



> 4、播放页的信息显示，最好就是 直接 3S 倒计时。我们普通用户都看不懂也不看这。



> 5、搜索页面，勾选一个搜索类别。如果不勾选类别会导致搜索一个资源很久。 

目前搜索有过滤选项,可以选择搜索特定类型，点击搜索框右边猫爪图标。
全局搜索需要改进，以后会支持选择特定规则进行搜索，类型选项会放在一起

> 6、自动暂时关闭站点，检测到报错。或者添加一个一件检测全部站点功能。把失效的坚持后默认都关闭。

管理站点有校验,失效的会被禁用，但是在某些情况下并不准确

**Reply by SONGjiemo** [reply]
是的，牛逼了.响应及时.  
现在就磁力播放 支持bt种子和磁力失败提示.
我看很多都是基于迅雷的那个内核开发的，别的基本上都是播放不出来
